api/authentication/router.js

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+const router = express.Router();
+import authenticate from './authenticate.js';
```

Comment:
Fix import 

Steps:
1. change the name of file api/authentication/controller/controller.js to api/authentication/controller/authenticate.js
2. change import in api/authentication/router.js
 import authenticate from './authenticate.js'; ->  import authenticate from './controller/authenticate.js'